Update visual-editor-interface.md (#6977)
dbt-labs/docs.getdbt.com: website/docs/docs/cloud/visual-editor-interface.md

title: "Navigate the interface" 
id: visual-editor-interface      
sidebar_label: "Navigate the interface" 

> description: "The visual editor interface contains an operator toolbar, operators, and a canvas to help you access and transform data through a seamless drag-and-drop experience in dbt Cloud." 

Reviewer: @rcg798 I think since we stress that this is VEs ultimate end goal is a model (even if the user doesn't know what that is at the start of their journey) that we keep that language in the description to optimize SEO. How about if we combine the two ideas:
```suggestion
description: "The Visual Editor interface contains an operator toolbar, operators, and a canvas to help you access and transform data through a seamless drag-and-drop dbt model creation experience in dbt Cloud." 
```

# Navigate the interface <Lifecycle status='beta'/> 

<p style={{ color: '#717d7d', fontSize: '1.1em' }}>

> The visual editor interface contains an operator toolbar, operators, canvas, built-in AI, and more to help you access and transform data through a seamless drag-and-drop experience in dbt Cloud. We do this through dbt models.

Reviewer: Same feedback as above:
```suggestion
The visual editor interface contains an operator toolbar, operators, canvas, built-in AI, and more to help you access and transform data through a seamless drag-and-drop dbt model creation experience in dbt Cloud.
```

## Canvas

> The visual editor has a sleek drag-and-drop canvas interface that allows you to create or modify dbt SQL models. It's like a digital whiteboard space that allows anyone to view and deliver trustworthy data. Use the canvas to:

Reviewer: It might be wise to avoid "anyone" because they still have to have a dbt license. It's not a tool anyone can access via a link share or be granted access with just an email address (like a shared Google Sheet). Also, I'm not a fan of "allows," which I put in there originally (that's on me). How about: 
```suggestion
The Visual Editor has a sleek drag-and-drop canvas interface for creating and modifying dbt SQL models. It's like a digital whiteboard space for easily viewing and delivering trustworthy data. Use the canvas to:
```

Author: Good point! As long as we aren't equating it to analysts I am happy : )
